internal/repository/postgresdb/user.go

```diff
@@ -430,9 +430,10 @@ func (us *userStore) GetUserById(ctx context.Context, reqData dto.GetUserByIdReq
 
 func (us *userStore) GetTop10Users(ctx context.Context, quarterTimestamp int64) (users []repository.Top10Users, err error) {
 
-	getTop10UserQuery := `select users.id, users.first_name, users.last_name, users.profile_image_url, sum(appreciations.total_reward_points) as AP from users join appreciations on users.id = appreciations.receiver group by users.id, appreciations.receiver order by AP desc limit 10`
+	afterTime := GetQuarterStartUnixTime()
```

Comment:
this has been sent already, no need to create again.

Reply:
removed afterTime